usaspending_api/references/v2/views/city.py

```diff
@@ -122,7 +121,46 @@ def create_es_search(scope, search_text, country=None, state=None):
             }
         ]
     }
-    return build_es_city_query(query, False, scope, country, state)
+    if country != "USA":
+        # A non-USA selected country
+        if country != "FOREIGN":
```

Comment:
This is the value captured by ALL_FOREIGN_COUNTIRES. We should make this reference the same variable to help if this gets refactored.